docs/markdown/Dependencies.md

```diff
@@ -849,6 +849,32 @@ version.
 
 *New in 0.54.0* the `system` method.
 
+## DIA SDK
```

Comment:
Somewhere (whatever the other dependencies are doing): `*New in 1.6.0*`

Reply:
Done in https://github.com/mesonbuild/meson/pull/13638/commits/52eeb494029131deb773eaf26c04cdcf257d46fb